sdcm/monitorstack/__init__.py

```diff
@@ -350,10 +366,11 @@ def restore_annotations_data(monitoring_stack_dir):
 
 
 @retrying(n=3, sleep_time=5, message='Start docker containers')
-def start_dockers(monitoring_dockers_dir, monitoring_stack_data_dir, scylla_version):  # pylint: disable=unused-argument
-    graf_port = GRAFANA_DOCKER_PORT
-    alert_port = ALERT_DOCKER_PORT
-    prom_port = PROMETHEUS_DOCKER_PORT
+def start_dockers(monitoring_dockers_dir, monitoring_stack_data_dir, scylla_version, tenants_number):  # pylint: disable=unused-argument
+    graf_port = get_free_port(ports_to_try=(GRAFANA_DOCKER_PORT + i for i in range(tenants_number)))
+    alert_port = get_free_port(ports_to_try=(ALERT_DOCKER_PORT + i for i in range(tenants_number)))
```

Comment:
@vponomaryov has a point, the ranges would be overlapping.

Reply:
let try to have a range of 50 ports each, which isn't overlapping

Reply:
changed default alert_docker_port to `6000`